uber/tests/models/test_merch.py

```diff
@@ -141,6 +141,7 @@ def test_shirt_info_marked_after_deadline(self, monkeypatch):
 class TestMerch:
     @pytest.fixture(autouse=True)
     def defaults(self, monkeypatch):
+        monkeypatch.setattr(c, 'SEPARATE_STAFF_MERCH', True)
```

Comment:
Ideally I'd take the time to write a separate set of tests for the case where ``SEPARATE_STAFF_MERCH`` was turned off to ensure we don't break that functionality for other events.  That didn't seem worth the time here, especially since I suspect a lot of the merch stuff may get redone down the road, but I can go in and add that if anyone thinks it's necessary.